openadapt/ml/finetune.py

```diff
@@ -0,0 +1,90 @@
+from openadapt.crud import *
+from openadapt import models, utils
+from copy import deepcopy
+import json
+from loguru import logger
+
+from openadapt.strategies.stateful import get_window_state_diffs
+
+
+def condense_window_state(recording_id: int):
+    """
+    Takes in a recording as input and outputs a list of tuples
+    of action, window events where window events are condensed
+    to contain basic information, as in stateful_test.py
+    """
+    grab_recording = get_recording_by_id(recording_id)
+    total_acx = get_action_events(grab_recording)
+
+    for i in range(len(total_acx)):
+        processed_acx, processed_wd = (
+            sanitize(total_acx[i])[0],
+            sanitize(total_acx[i])[1],
+        )
+        processed_wd.pop("meta")
+        # task_description = recording.task_description
+
+        logger.debug(f"{total_acx[i].timestamp=}")
+        logger.debug(f"{total_acx[i].window_event_timestamp=}")
```

Comment:
I have the infrastructure ready to finetune on bare-bones recordings (see below) but I'm curious about which timestamp we intend to finetune on, as the action and window event for each pair seem to have different timestamps.
```
2023-08-04 01:06:47.707 | DEBUG    | __main__:condense_window_state:27 - total_acx[i].timestamp=1693194018.9387414
2023-08-04 01:06:47.707 | DEBUG    | __main__:condense_window_state:30 - total_acx[i].window_event_timestamp=1693194018.6880429
```

Reply:
For fine-tuning I think we want to ignore the timestamp for now in the name of simplicity. What do you think?